chart/templates/secrets/git-ssh-key-secret.yaml

```diff
@@ -0,0 +1,34 @@
+{{/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+*/}}
+
+{{- if and .Values.dags.gitSync.gitSshKey .Values.dags.gitSync.enabled}}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}
+    heritage: {{ .Release.Service }}
+    {{- with .Values.labels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: airflow-ssh-secret
```

Comment:
This name need to be updated to use `airflow.fullname`, like this: https://github.com/apache/airflow/blob/93ba263732a30ed51fdae0b2bff7cd959be115b8/chart/templates/secrets/webserver-secret-key-secret.yaml#L28

You'll also want to this secret to be used automatically (as in, don't require the user to configure the key name too). It'll be similar to this: https://github.com/apache/airflow/blob/93ba263732a30ed51fdae0b2bff7cd959be115b8/chart/templates/_helpers.yaml#L390-L392
You'll likely need a bit more logic where the key is mounted though.

Reply:
Thanks, change it like this given example